src/components/verification-grid/verification-grid.ts

```diff
@@ -933,6 +1054,158 @@ export class VerificationGridComponent extends WithShoelace(AbstractComponent(Li
     this.currentSubSelection = gridTiles.filter((tile) => tile.selected).map((tile) => tile.model);
   }
 
+  /**
+   * A common method that can be used to create consistent selection behavior
+   * across the different selection methods (click, alt, arrow, tab & highlight)
+   */
+  private processSelection(
+    tileIndices: number | number[],
+    { additive = false, toggle = false, range = false, focus = false }: SelectionOptions = {},
+  ): void {
+    if (!this.canSubSelect()) {
+      return;
+    }
+
+    this.isCurrentAutoSelected = false;
+
+    let selectionBehavior = this.selectionBehavior;
+    if (selectionBehavior === "default") {
+      selectionBehavior = this.isMobileDevice() ? "tablet" : "desktop";
+    }
+
+    const selectionIndex = tileIndices;
+
+    // in desktop mode, unless the ctrl key is held down, clicking an element
+    // removes all other selected items
+    // while it is not possible to press the ctrl key on a tablet, the user can
+    // still overwrite the selection behavior using the selection-behavior
+    // attribute. Therefore, we have to check that we are not on explicitly
+    // using tablet selection mode.
+    if (selectionBehavior === "desktop" && !toggle && !additive && !focus) {
+      this.removeSubSelection();
+    }
+
+    const iterableIndices = Array.isArray(selectionIndex) ? selectionIndex : [selectionIndex];
+    for (const tileIndex of iterableIndices) {
+      // If the "focus" selection behavior is set, we want to focus the tile but
+      // not select it.
+      if (focus) {
+        this.focusTile(tileIndex);
+        this.focusHead = tileIndex;
```

Comment:
is there a reason focusTile doesn't set focusHead?

Reply:
Good catch, it does, so this assignment was a boop